src/database/queries/proposals/insertOne.sql

```diff
@@ -5,8 +5,4 @@ INSERT INTO proposals (
   :externalId,
   :opportunityId
 )
-RETURNING
-  id as "id",
-  external_id AS "externalId",
-  opportunity_id AS "opportunityId",
-  created_at AS "createdAt"
+RETURNING proposal_to_json(proposals) AS "object";
```

Comment:
If I'm reading this correctly, this means an `INSERT` now requires `JOIN`s. On the one hand I am surprised by that . On the other hand it may be what we want.

Reply:
The main thinking here is that our resulting entity would look the same as it would look if accessed via a `GET` / would represent the entity state.  I *think* it's at least close enough to what we want that it would be valuable (otherwise a client would need to make a followup `GET` to get a full object after `POST`, which feels un-REST-ful)